modules/openapi-generator/src/main/resources/rust-server/client-mod.mustache

```diff
@@ -246,18 +247,23 @@ impl<F, C> Api<C> for Client<F> where
     C: Has<XSpanIdString> {{#hasAuthMethods}}+ Has<Option<AuthData>>{{/hasAuthMethods}}{
 {{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}
     fn {{#vendorExtensions}}{{{operation_id}}}{{/vendorExtensions}}(&self{{#allParams}}, param_{{{paramName}}}: {{^required}}Option<{{/required}}{{#isListContainer}}&{{/isListContainer}}{{{dataType}}}{{^required}}>{{/required}}{{/allParams}}, context: &C) -> Box<Future<Item={{{operationId}}}Response, Error=ApiError>> {
+        let mut query_string = self::url::form_urlencoded::Serializer::new("".to_owned());
 {{#queryParams}}{{#-first}}
         // Query parameters
```

Comment:
Could we perhaps move the `let mut query_string = ...` line inside the `{{#queryParams}}{{#-first}}` tags? That would allow us to remove the checks below for `query_string_str != ""` in the case where we don't have any query strings.